docs/src/main/scala/bloop/Docs.scala

```diff
@@ -1,12 +1,27 @@
 package bloop
 
+import bloop.docs.{ReleasesModifier, Sonatype}
+import mdoc.MainSettings
+
+import scala.meta.io.AbsolutePath
+
 object Docs {
   def main(args: Array[String]): Unit = {
-    val settings = mdoc.MainSettings()
-      .withSiteVariables(Map("VERSION" -> bloop.internal.build.BuildInfo.version))
+    val cwd0 = AbsolutePath.workingDirectory
+    // Depending on who runs it (sbt vs bloop), the current working directory is different
+    val cwd = if (!cwd0.resolve("docs").isDirectory) cwd0.toNIO.getParent else cwd0.toNIO
+
+    val settings = MainSettings()
+      .withSiteVariables(Map(
+        "VERSION" -> Sonatype.release.version,
+        "LATEST_VERSION" -> bloop.internal.build.BuildInfo.version
+      ))
       .withArgs(args.toList)
+      // it should work with mdoc when run inside bloop but it doesn't, let's wait until it's fixed
+      .withWorkingDirectory(cwd)
```

Comment:
It will work if you update `withOut` instead of `withWorkingDirectory`.

I opened https://github.com/olafurpg/mdoc/pull/111 updating `withWorkingDirectory` to also update `docs` and `out`.